src/app/auth/callback/route.ts

```diff
@@ -0,0 +1,19 @@
+import { createRouteHandlerClient } from '@supabase/auth-helpers-nextjs'
+import { cookies } from 'next/headers'
+import { NextResponse } from 'next/server'
+
+import type { NextRequest } from 'next/server'
+import type { Database } from '@/app/auth/lib/database.types'
+
+export async function GET(request: NextRequest) {
```

Comment:
what's the idea to name this function GET? name should describe what kind of action is being happened.

Reply:
I've tried to change the name of this function, but it's returned 'error'. I think the 'superbase' needs this kind of function...